doc/source/_static/thumbnails/resize.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+
+from PIL import Image
+
+
+def resize_with_background(input_image_path, output_image_path, target_size):
+    """Resize an image while maintaining aspect ratio and centering it on a white background.
+
+    Parameters
+    ----------
+    input_image_path : Path
+        The path to the input image.
+    output_image_path : Path
+        The path to save the output image.
+    target_size : tuple[int, int]
+        The target size of the output image as a tuple (width, height) in pixels.
+    """
+    # Open the input image
+    img = Image.open(input_image_path).convert("RGBA")  # Ensure the image has an alpha channel
+
+    # Resize the image while maintaining aspect ratio
+    img.thumbnail(target_size, Image.LANCZOS)
+
+    # Create a white background of the target size
+    background = Image.new(
+        "RGBA", target_size, (255, 255, 255, 255)
```

Comment:
```suggestion
        "RGBA", target_size, (255, 255, 255, 0)
```
why not initially remove the alpha value making the entire background initially transparent.

Reply:
We need to make it white - i.e. remove transparency. This is on purpose.